api/src/services/usergroup/usergroup.test.ts

```diff
@@ -273,8 +275,101 @@ describe('\'usergroup\' service', () => {
 
     const userGroupStillExist = await expect(app.service('usergroup').get(`${userGroup.user_id},${userGroup.group_id}`)).rejects.toThrowError(NotFound) as undefined | Usergroup
 
-    // Clean the database whether the test succeeds or not
-    if (userGroupStillExist) await app.service('usergroup').remove(`${userGroup.user_id as number},${userGroup.group_id as string}`)
+    expect(userGroupStillExist).toBeUndefined()
+
+    // Clean the database
+    if (user) await app.service('user').remove(user.id)
+  })
+
+  it('verify a MEMBER user with OWNER role can assign user to group', async () => {
```

Comment:
Those tests are great,

You could "refacto" those tests by adding `describe` in this section, this would allow you to have a `beforeAll` to put inside the `patch` of the lambda role, and a `afterAll` to restore initial role.

Could you add also tests for the `ADMIN` role ? (actually, checks need to be the same for usergroup, so copy/paste your code, and change the `OWNER` role by `ADMIN` one)

Reply:
Done